README.md

```diff
@@ -97,50 +96,50 @@ Premade boilerplates for Fiber. **(WIP)**
 - [embedmode/fiberseed](https://github.com/embedmode/fiberseed)
 
 
-## Recipes
+## 📁 Recipes
 Recipes for Fiber. **(WIP)**
-- https://github.com/gofiber/recipes
+- [recipes](https://github.com/gofiber/recipes) - Official Fiber cookbook
 - https://github.com/kiyonlin/fiblar-demo
 - https://github.com/koddr/tutorial-go-fiber-rest-api
 - https://github.com/firebase007/go-rest-api-with-fiber
 - https://github.com/embedmode/fiberseed
 - https://github.com/chawk/go_fiber_quickstart
 - https://github.com/EricLau1/go-fiber-auth-api
 
-## Tools
+## 🛠️ Tools
 Several tools to make Fiber usage easier. **(WIP)**
-- https://github.com/gofiber/cli
-- https://github.com/go-dawn/dawn
-- https://github.com/tompston/gomakeme
+- [cli](https://github.com/gofiber/cli) - Fiber Command Line Interface
```

Comment:
I think we should write: go-dawn/dawn, gofiber/cli, tompston/gomakeme instead of cli, dawn, gomakeme.